### ISSUE TITLE
latexindent.aux is missing

### ISSUE THREAD

**Issue by albertodiazdorado** [cmhughes/latexindent.pl#135]
The update procedure fails to find the document latexindent.aux, which does not exist. After `git pull`ing the latest commit, I follow the instructions in helper-scripts/readme.txt. First, I run:

`perl documentation-default-settings-update.pl`

which runs fine. But then, running:

`perl documentation-default-settings-update.pl -r`

fails and prompts: 

> Could not open input file, latexindent.aux at documentation-default-settings-update.pl line 93.

There is actually no `latexindent.aux` file in the repository. Shall it read `latexindent.tex` instead?

**Comment by cmhughes** [comment]
Hello!

I'm not clear what you're trying to achieve here; are you wanting to update
the documentation?

There should be no need to do anything with the documentation folder
scripts....

Let me know
Chris

**Comment by albertodiazdorado** [comment]
Thank you for your answer. To be fair, what I am trying to do is to update the package. What I normally do with packages installed via GIT is `git pull` and then `makepkg`, but this package works a little bit differently since there is no `PKGBUILD`.

So I tried to run `update-version.sh`:

`cd helper-scripts
./update-version.sh`

but it does not work and prompts the same message that I quoted above:

> Could not open input file, latexindent.aux at documentation-default-settings-update.pl line 93.

It turns out that `update-version.sh` calls `documentation-default-settings.pl`, hence the error. I tried to locate the error more precisely in my original comment, but what I really want to achive is to install the new version after a `git pull`
